### PR TITLE
new playbook: asreview

### PR DESCRIPTION
**Pull request by SytseGroenwold**
My first pull request! 
I have (and for a little while will) include you both as reviewer, because 6 see more than 4, right?
Jelle and I have tested the functionality of the workspace you can create with this, and it works fine. Multi-user support is not present in ASReview yet and the things Jelle and I have tried did not lead to a successful result, so the plugin will remain for a single-user workspace.

### CODE REVIEW

**Review by tsmeele — CHANGES_REQUESTED** [review]
see comments

File: docs/playbooks/asreview.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Playbook <name>
+[back to index](../index.md#Playbooks)
+
+## Summary
+Automatic Systematic Review (ASReview) is an application that uses active machine learning to facilitate and predict relevancy of papers during a systematic review process.
```
Comment (J535D165) [inline docs/playbooks/asreview.md:5]:
```suggestion
ASReview LAB is an application that uses active learning to facilitate and predict the relevancy of papers during a systematic review.
```

File: docs/roles/asreview.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Role <name>
+[back to index](../index.md#Roles)
+
+## Summary
+Automatic Systematic Review (ASReview) is an application that uses active machine learning to facilitate and predict relevancy of papers during a systematic review process.
```
Comment (J535D165) [inline docs/roles/asreview.md:5]:
```suggestion
ASReview LAB is an application that uses active learning to facilitate and predict the relevancy of papers during a systematic review.
```

File: docs/roles/asreview.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+## Summary
+Automatic Systematic Review (ASReview) is an application that uses active machine learning to facilitate and predict relevancy of papers during a systematic review process. 
+For more information, visit the [official website](https://asreview.nl/).
+
+## Requires
+* Desktop environment
+* Has been tested on Ubuntu 20.04
+
+## Description
+This role installs ASReview through pip, as recommended by the documentation.
```
Comment (J535D165) [inline docs/roles/asreview.md:13]:
```suggestion
This role installs ASReview LAB through pip, as recommended by the documentation.
```

File: docs/roles/asreview.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Role <name>
+[back to index](../index.md#Roles)
+
+## Summary
+Automatic Systematic Review (ASReview) is an application that uses active machine learning to facilitate and predict relevancy of papers during a systematic review process. 
+For more information, visit the [official website](https://asreview.nl/).
```
Comment (J535D165) [inline docs/roles/asreview.md:6]:
```suggestion
For more information, visit https://asreview.ai/.
```

File: roles/asreview/tasks/main.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+- name: Install ASReview through pip
```
Comment (J535D165) [inline roles/asreview/tasks/main.yml:2]:
```suggestion
- name: Install ASReview LAB through pip
```

File: asreview.yml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
+- name: Install ASReview system-wide
```
Comment (J535D165) [inline asreview.yml:2]:
```suggestion
- name: Install ASReview LAB system-wide
```

**Review by J535D165 — COMMENTED** [review]
Nice! Some small textual/branding suggestions.

File: docs/playbooks/asreview.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# Playbook <name>
```
Comment (tsmeele) [inline docs/playbooks/asreview.md:1]: pls add name

Comment (SytseGroenwold) [reply docs/playbooks/asreview.md:1]: Test.

Comment (tsmeele) [inline docs/playbooks/asreview.md:1]:
```suggestion
# Playbook ASReview
```

**Review by tsmeele — APPROVED** [review]

**Review by J535D165 — APPROVED** [review]

**Pull request merged**